pkgs/by-name/dr/druid/package.nix

```
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, extensions ? { }
+, libJars ? [ ]
+, nixosTests
+, mysqlSupport ? true
+}:
+let
+  inherit (lib) concatStringsSep mapAttrsToList optionalString forEach;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache-druid";
+  version = "30.0.0";
+  src = fetchurl {
+    url = "https://dlcdn.apache.org/druid/${finalAttrs.version}/apache-druid-${finalAttrs.version}-bin.tar.gz";
+    hash = "sha256-mRYorVkNzM94LP53G78eW20N5UsvMP7Lv4rAysmPwXw=";
+  };
+  mysqlConnector = fetchurl {
+    url = "https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.48/mysql-connector-java-5.1.48.jar";
+    hash = "sha256-VuJsqqOCH1rkr0T5x09mz4uE6gFRatOAPLsOkEm27Kg=";
+  };
+  dontBuild = true;
+  passthru = {
```

```suggestion
  version = "30.0.0";

  src = fetchurl {
    url = "https://dlcdn.apache.org/druid/${finalAttrs.version}/apache-druid-${finalAttrs.version}-bin.tar.gz";
    hash = "sha256-mRYorVkNzM94LP53G78eW20N5UsvMP7Lv4rAysmPwXw=";
  };

  mysqlConnector = fetchurl {
    url = "https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.48/mysql-connector-java-5.1.48.jar";
    hash = "sha256-VuJsqqOCH1rkr0T5x09mz4uE6gFRatOAPLsOkEm27Kg=";
  };

  dontBuild = true;

  passthru = {
```

Please place some new lines to make the code readable.